zavod/zavod/logic/pep.py

```diff
@@ -153,7 +153,7 @@ def occupancy_status(
     categorisation: Optional[PositionCategorisation] = None,
 ) -> Optional[OccupancyStatus]:
     if death_date is not None and death_date < backdate(current_time, AFTER_DEATH):
```

Comment:
since the original string is returned if a date doesn't parse, we probably want to

1. start warning if dates don't parse
2. and eventually parse and compare them as prefixdates instead of ISO strings.

Currently we can be comparing a date string to an unsuccessfully-parsed date

Reply:
doesn't have to be in this change